webapp/src/components/AssessmentPage.tsx

```diff
@@ -0,0 +1,473 @@
+    return <TableRow>{children}</TableRow>;
+  }
+  switch (status) {
+    case 'Active':
+      return statusTab === StatusTab.Active ? (
+        <TableRow>{children}</TableRow>
+      ) : null;
+    case 'Upcoming':
+      return statusTab === StatusTab.Upcoming ? (
+        <TableRow>{children}</TableRow>
+      ) : null;
+    case 'Submitted':
+    case 'Graded':
+    case 'Unsubmitted':
+    default:
+      return statusTab === StatusTab.Past ? (
+        <TableRow>{children}</TableRow>
+      ) : null;
+  }
+}
+
+function renderButtonByStatus(status: string) {
+  switch (status) {
+    case 'Active':
+      return (
+        <Button variant="contained" size="small">
+          Start
+        </Button>
+      );
+    case 'Submitted':
+    case 'Graded':
+      return (
+        <Button variant="contained" size="small">
+          View
+        </Button>
+      );
+    case 'Upcoming':
+    case 'Unsubmitted':
+    default:
+      return null;
+  }
+}
+
+function renderChipByStatus(status: string) {
+  switch (status) {
+    case 'Active':
+      return (
+        <Chip
+          variant="outlined"
+          color="primary"
+          size="small"
+          icon={<ScheduleOutlinedIcon />}
+          label="Active"
+        />
+      );
+    case 'Submitted':
+      return (
+        <Chip
+          variant="outlined"
+          color="primary"
+          size="small"
+          icon={<CheckOutlinedIcon />}
+          label="Submitted"
+        />
+      );
+    case 'Graded':
+      return (
+        <Chip
+          variant="outlined"
+          color="success"
+          size="small"
+          icon={<DoneAllOutlinedIcon />}
+          label="Graded"
+        />
+      );
+    case 'Upcoming':
+      return (
+        <Chip
+          variant="outlined"
+          size="small"
+          icon={<LockClockOutlinedIcon />}
+          label="Upcoming"
+        />
+      );
+    case 'Unsubmitted':
+      return (
+        <Chip
+          variant="outlined"
+          color="error"
+          size="small"
+          icon={<CancelOutlinedIcon />}
+          label="Unsubmitted"
+        />
+      );
+    default:
+      return null;
+  }
+}
+
+const StyledBadge = styled(Badge)<BadgeProps>(({ theme }) => ({
+  '& .MuiBadge-badge': {
+    border: `2px solid ${theme.palette.background.paper}`,
+    padding: '0 4px',
+  },
+}));
+
+const AssessmentPage = () => {
+  const [currentStatusTab, setCurrentStatusTab] = useState(StatusTab.Active);
+  const handleChangeTab = (
+    event: React.SyntheticEvent,
+    newCurrentStatusTab: number
+  ) => {
+    setCurrentStatusTab(newCurrentStatusTab);
+  };
+
+  return (
+    <Container>
+      <Stack
+        direction={{ xs: 'column', md: 'row' }}
+        justifyContent="space-between"
+        alignItems={{ xs: 'flex-start', md: 'center' }}
+      >
+        <h1>Program Assessments</h1>
+      </Stack>
+
+      <Box sx={{ width: '100%', bgcolor: 'background.paper' }}>
```

Comment:
You should handle the case where the list of assessments is empty and not display the tabs or the table and have a message instead; something along the lines of "You have no available assessments at this time."